### DISCUSSION TITLE
Custom button on list view

### DISCUSSION THREAD

**Discussion by malikovss** [aminalaee/sqladmin#376] [Q&A] [answered]
Hi, is it possible to add custom button as a column on list view row?

I tried
column_list = [..., 'test']
column_formatters = {'test': lambda m, a: "<a>button</a>"}

And got attribute error.

**Comment by aminalaee** [comment, marked as answer]
Hi that would be custom actions I think.
It’s not ready yet but I plan to do it in the next release.

**Reply by malikovss** [reply]
Got it. Thanks
